govc/library/secpol/ls.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package secpol
```

Comment:
The command name is `policy`, can you rename the package to match?  I think we have a few places where package + command name don't match, and it's just cosmetic, but more consistent for the names to be the same.

Reply:
Makes sense, will do.